scripts/release_scripts/run_macaron.sh

```diff
@@ -314,25 +307,24 @@ if [[ -n "${sbom_path}" ]]; then
     mounts+=("-v" "${sbom_path}:${MACARON_WORKSPACE}/sbom/${file_name}:ro")
 fi
 
-# MACARON entrypoint - Verify action argvs
-# Determine the provenance path to be mounted into ${MACARON_WORKSPACE}/provenance/${file_name}
-# This is for macaron verify action.
-if [[ -n "${arg_provenance}" ]]; then
-    provenance="${arg_provenance}"
-    err=$(check_file_exists "${provenance}" "-pr/--provenance")
+# Determine the provenance expectation path to be mounted into ${MACARON_WORKSPACE}/prov_expectations/${file_name}
+if [[ -n "${arg_prov_exp}" ]]; then
+    prov_exp="${arg_prov_exp}"
+    err=$(check_file_exists "${prov_exp}" "-pe/--provenance-expectation")
     if [[ -n "${err}" ]]; then
         echo "${err}"
         exit 1
     fi
-    file_name="$(basename "${provenance}")"
-    argv_action+=("--provenance" "${MACARON_WORKSPACE}/provenance/${file_name}")
+    file_name="$(basename "${prov_exp}")"
+    argv_action+=("--provenance-expectation" "${MACARON_WORKSPACE}/prov_expectations/${file_name}")
 fi
-if [[ -n "${provenance}" ]]; then
-    provenance="$(ensure_absolute_path "${provenance}")"
-    mounts+=("-v" "${provenance}:${MACARON_WORKSPACE}/provenance/${file_name}:ro")
+if [[ -n "${prov_exp}" ]]; then
+    prov_exp="$(ensure_absolute_path "${prov_exp}")"
+    mounts+=("-v" "${prov_exp}:${MACARON_WORKSPACE}/prov_expectations/${file_name}:ro")
 fi
 
-# POLICY_ENGINE entrypoint.
+# MACARON action - verify-policy
```

Comment:
```suggestion
# MACARON entrypoint - verify-policy action argvs
```
So it would be consistent with other actions.

Reply:
See commit [c09c5a3](https://github.com/oracle-samples/macaron/pull/192/commits/c09c5a3379ef0733cd926c0b767480d77809a220).